Fix to allow custom `CheckpointIO` with strategy classes (#13785)
Lightning-AI/lightning: src/pytorch_lightning/CHANGELOG.md

- Fixed `Trainer.predict(return_predictions=False)` to track prediction's batch_indices ([#13629](https://github.com/Lightning-AI/lightning/pull/13629))

> - Fixed and issue to allow custom `CheckpointIO` plugins with strategies ([#13785](https://github.com/Lightning-AI/lightning/pull/13785))

Reviewer: ```suggestion
- Fixed and issue that prevented setting a custom `CheckpointIO` plugin with strategies ([#13785](https://github.com/Lightning-AI/lightning/pull/13785))
```
